bedrock/firefox/templates/firefox/dev-firstrun.html

```diff
@@ -70,14 +58,31 @@
 
 {% block content %}
 
-<main role="main" {% if l10n_has_tag('new_doorhanger') %}class="sync-reminder"{% endif %} >
+<main role="main" class="sync-reminder">
```

Comment:
If we're removing this l10n tag, I believe you can remove the class here as well. It was used as a conditional logic check in the page's JS here: https://github.com/mozilla/bedrock/blob/master/media/js/firefox/dev-firstrun.js#L279-L280

If we update this JS and always execute the code in the conditional there, then this class is no longer needed. Sorry if this seems confusing (it was a temporary hack to avoid having to wait on l10n).